Assets/MRTK/SDK/Experimental/MixedRealityKeyboard/MixedRealityKeyboardBase.cs

```diff
@@ -121,11 +152,19 @@ protected virtual void Start()
         private void OnInputPaneHiding(InputPane inputPane, InputPaneVisibilityEventArgs args)
         {
             OnKeyboardHiding();
+            if (DisableUIInteractionWhenTyping && inputModule != null)
+            {
+                inputModule.ProcessPaused = false;
```

Comment:
Do you think we can use the corresponding input system enable / disable functionality instead of adding this new "paused" state? We have the ability to temporarily pause / resume the input system:

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/a03e5e2e0c9e28c889c8df5c7436de506b59b0ce/Assets/MRTK/Services/InputSystem/MixedRealityInputSystem.cs#L726-L744

Reply:
I experimented with that function just now but believe there is a bug... The bug is after the input system is reactivated a gaze cursor appears and stays in the view despite hand based cursors are also present. So we might have to stick with the current implementation if we do not want to invest time in that bug right now.